test/src/org/exist/xmldb/XmldbURITest.java

```diff
@@ -13,7 +13,7 @@
 
 public class XmldbURITest {
 
-    private final static String jettyPort = System.getProperty("jetty.port");
+    private final static String jettyPort = System.getProperty("jetty.port", "8080");
```

Comment:
@dizzzz You can run this test in parallel without needing to make it configurable. It has no dependency on Jetty or an eXist server.